src/App.tsx

```diff
@@ -23,22 +23,27 @@ export interface IState {
 }
 
 export default class App extends React.Component <IProps, IState> {
+    private name: string;
 
     constructor(props: IProps) {
         super(props);
 
         const playerStats = new PlayerStats();
         const eventTracker = new EventTracker(
             [new BoomerGregorEvent()],
-            [new PickFacultyEvent()]
+            [new LandingEvent(), new PickFacultyEvent()]
         );
         let firstEvent = eventTracker.getNextEvent();
 
+        //TODO: should we auto allocate a dummy player name or allow user to input their own?
+        this.name = "P1";
```

Comment:
Capturing the player's name is out of the scope of this ticket, so I'm ok leaving this for a follow up PR.